content/en/docs/setup/install/_index.md

```diff
@@ -6,35 +6,72 @@ description: >
   Describes how to install and set up Spinnaker so that it can be configured for use in production.
 ---
 
-This section describes how to install and set up Spinnaker so that it can be configured for
-use in production. If you just want to evaluate Spinnaker without much work, one of the options
-in [Quickstart](/docs/setup/quickstart/) might be a better choice.
+This section describes how to install and set up Spinnaker so that it can be use in production. If
+you just want to evaluate Spinnaker without much work, one of the options in
+[Quickstart](/docs/setup/quickstart/) or a [Local installation](/docs/setup/install/local/) might be
+a better choice.  It's recommended that you review the  [spinnaker architecture](/docs/reference/architecture) 
+to understand how spinnaker operates before installation.
```

Comment:
I would simplify it more

Before installing, review Spinnaker's [architecture](/docs/reference/architecture)